### DISCUSSION TITLE
Discrepancy between uniqueMoleculeCount and uniqueMoleculeFraction

### DISCUSSION THREAD

**Discussion by rstubbins** [milaboratory/mixcr#1530] [Q&A]
Hi, I used mixcr to process data obtained from a dataset generated with the NEBNext mouse kit. I'm trying to plot the number of unique clones to demonstrate that my experimental condition has a small number of dominant clones vs. wt. I was going to plot the "uniqueMoleculeFraction" to show this, but I realized that this is discordant with the "uniqueMoleculeCount" and does not sum to 1. What am I missing here? Is this the right output parameter to plot?

**Comment by mizraelson** [comment]
Hi, what is the sum of values in your `uniqueMoleculeFraction` column? It should be `1`. Can you share the command you used for data analysis?
I think the best way is to first normalize your data to the same number of UMIs with [`mixcr downsample`](https://mixcr.com/mixcr/reference/mixcr-downsample/) and then plot clonotype abundance on a barplot with  `uniqueMoleculeFraction` for each clone in different colors.
